packages/frontend/src/timelineHeader.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-FileCopyrightText: syuilo and misskey-project
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { reactive } from 'vue';
+import { i18n } from '@/i18n.js';
+import { userListsCache } from '@/cache.js';
+import { isLocalTimelineAvailable, isGlobalTimelineAvailable } from '@/store.js';
+const lists = await userListsCache.fetch();
+export const timelineHeaderItemDef = reactive({
```

Comment:
```suggestion
export type TimelineHeaderItem = 
	'home' |
	'local' |
	'social' |
	'global' |
	'lists' |
	'antennas' |
	'channels' | 
	`list:${string}`

type TimelineHeaderItemsDef = {
	title: string;
	icon: string;
	iconOnly?: boolean; // わからん
}

export const timelineHeaderItemDef = reactive<Record<TimelineHeaderItem, TimelineHeaderItemsDef>>({
```

Typeを置いといて色んな場所で使うと後々楽そう